library/Ivoz/Ast/Domain/Model/Voicemail/VoicemailAbstract.php

```diff
@@ -541,7 +541,23 @@ public function getAlias(): ?string
     protected function setEmail(?string $email = null): static
     {
         if (!is_null($email)) {
-            Assertion::maxLength($email, 80, 'email value "%s" is too long, it should have no more than %d characters, but has %d characters.');
+            
+            Assertion::maxLength(
+                $email,
+                80,
+                'Email value "%s" is too long, it should have no more than %d characters, but has %d characters.'
+            );
+
+            Assertion::true(
+                !preg_match('/\s/', $email),
+                'Email address should not contain spaces.'
+            );
+
+            Assertion::regex(
+                $email,
+                '/^[^\s@]+@[^\s@]+\.[^\s@]+$/',
+                'Invalid email address format.'
+            );
```

Comment:
`Assertion::email($email, 'Invalid email address format')`

Validating emails is tricky. In my opinion, it's better to rely on a library for this. The assertion mentioned uses PHP's built-in functionality.

If we don't need to strictly enforce email format but simply want to prevent clients from providing more than one address, we could count the occurrences of the '@' symbol and accept only one.